charts/tractusx-connector-azure-vault/values.yaml

```diff
@@ -453,12 +453,12 @@ dataplane:
     # choice for the user. This also increases chances charts run on environments with little
     # resources, such as Minikube. If you do want to specify resources, uncomment the following
     # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-    # limits:
-    #   cpu: 100m
-    #   memory: 128Mi
-    # requests:
-    #   cpu: 100m
-    #   memory: 128Mi
+  limits:
+    cpu: 1.5
+    memory: 512Mi
```

Comment:
This is different in regard to the other helm charts

Reply:
Has been adjusted.